docs/config.rst

```diff
@@ -46,7 +46,12 @@ core
 
     If ``true``, Gordon will continue running even if installed & configured plugins can not be loaded. Plugin exceptions will be logged as warnings with tracebacks.
 
-    If ``false``, Gordon will exit out if it can't load one or more plugins.
+    If ``false``, Gordon will exit out if it can't load one or more plugins
```

Comment:
Nitpick: I think the final `.` was removed here.

Reply:
hmm.. will fix.